docs/advanced/peering/advanced-offloading.md

```diff
@@ -0,0 +1,251 @@
+# Offloading in depth
+
+## Overview
+
```

Comment:
I would add a note telling that this is an advanced proceed and to stick with `liqoctl offload` when there aren't specific needs. (Like, for example, we have in [this page](https://github.com/liqotech/liqo/blob/docs/docs/advanced/peering/inter-cluster-authentication.md?plain=1#L18))